### ISSUE TITLE
Add currently running testcase to status bar

### ISSUE THREAD

**Issue by Pilchie** [Pilchie/xunit.runner.wpf#29]

**Comment by DustinCampbell** [comment]
Yeah, I was looking at that. It'll make the communication with the worker process (i.e. also tell us when a test starts running), but shouldn't be too big of a deal.

**Comment by Pilchie** [comment]
It could be throttled to only report back once every n milliseconds, etc if it's a perf issue.

**Comment by DustinCampbell** [comment]
I doubt it would be. Passing a single string through a named pipe seems more efficient than actually running the tests. :smile:
